classes/war/WarManager.php

```diff
@@ -457,31 +482,41 @@ public static function getPlayerAttackOrDefendRaidTargets(System $system, User $
         $oldest_active_raid_time = (microtime(true) * 1000) - $max_raid_duration_ms;
 
         $result = $system->db->query("SELECT
-            `operations`.`user_village` as `attacking_user_village`, 
+            `operations`.`user_village`, 
             `operations`.`target_id`,
+            `operations`.`target_village`,
             `region_locations`.`x`, 
             `region_locations`.`y`, 
             `region_locations`.`map_id`, 
             `region_locations`.`name` 
             FROM `operations`
             INNER JOIN `region_locations` ON `region_locations`.`id` = `operations`.`target_id`
-            WHERE (`operations`.`user_village` = {$player->village->village_id} OR `operations`.`target_village` = {$player->village->village_id})
             AND `user_id` != {$player->user_id}
             AND `last_update_ms` > {$oldest_active_raid_time}
             AND `status` = " . Operation::OPERATION_ACTIVE . " 
             AND `operations`.`type` = " . Operation::OPERATION_RAID . "
-            GROUP BY `operations`.`target_id`, `attacking_user_village`");
+            GROUP BY `operations`.`target_id`");
```

Comment:
```suggestion
            GROUP BY `operations`.`target_id`, `operations`.`target_village`");
```

Reply:
This should functionally be the same (as there will not be multiple values of `target_village` for a given `target_id`) and makes SQL happy with `only_full_group_by` mode enabled